docs/modules/getting_started/pages/basic_walkthrough.adoc

```diff
@@ -77,10 +45,7 @@ See the https://grafana.com/docs/grafana/latest/datasources/[official Grafana do
 Follow these steps to set up a datasource:
 
 . Navigate to the home page of your Grafana instance.
-. In the left navigation menu, select menu:Configuration[Data Sources]:
-+
-image::gf-data-sources.png["Grafana menu displaying navigation options. The Data Sources option is highlighted.", 250]
-
+. In the left navigation menu, select menu:Administration[Data ources].
 . Select *Add Data Source*.
 . In the *Search* box, type *OpenNMS Entities*.
 . Select the *OpenNMS Entities* datasource.
```

Comment:
User will want to repeat the same steps with a Performance datasource, and optionally a Flow datasource.
Personally, I would suggest stepping through a Perf DS first, as that would become the default datasource